reinforce/training/replay_buffer.py

```
@@ -0,0 +1,561 @@
+                # ##>: Sample batch using selected method.
+                if self._use_vectorized:
+                    batch, weights = self._buffer.sample_batch_vectorized(self._batch_size)
+                else:
+                    batch, weights = self._buffer.sample_batch(self._batch_size)
+
+                # ##>: Put in queue (blocks if full).
+                self._queue.put((batch, weights), timeout=1.0)
+
+            except Exception:
+                # ##>: Silently continue on errors to keep thread alive.
+                continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)